### ISSUE TITLE
🐛 Unable to retrieve the JNI environment. Is the Thread attached?

### ISSUE THREAD

**Issue by mrousavy** [cuvent/react-native-vision-camera#456]
### What were you trying to do?

Running a Frame Processor for a long time.

### Reproduceable Code

```tsx
Example App.
```

### What happened instead?

It hard-crashes.

### Relevant log output

```shell
2021-09-24 14:05:20.225 20583-20583/? A/DEBUG: Build fingerprint: 'HUAWEI/VTR-L29/HWVTR:9/HUAWEIVTR-L29/9.1.0.275C432:user/release-keys'
2021-09-24 14:05:20.226 20583-20583/? A/DEBUG: Revision: '0'
2021-09-24 14:05:20.226 20583-20583/? A/DEBUG: ABI: 'arm64'
2021-09-24 14:05:20.227 20583-20583/? A/DEBUG: Happend: 'Fri Sep 24 14:05:20 2021
    '
2021-09-24 14:05:20.227 20583-20583/? A/DEBUG: SYSVMTYPE: Art
    APPVMTYPE: Art
2021-09-24 14:05:20.227 20583-20583/? A/DEBUG: pid: 20436, tid: 20507, name: hades  >>> com.mrousavy.camera.example <<<
2021-09-24 14:05:20.227 20583-20583/? A/DEBUG: signal 6 (SIGABRT), code -6 (SI_TKILL), fault addr --------
2021-09-24 14:05:20.227 20583-20583/? A/DEBUG: Abort message: 'terminating with uncaught exception of type std::runtime_error: Unable to retrieve jni environment. Is the thread attached?'
2021-09-24 14:05:20.230 20436-20515/com.mrousavy.camera.example I/GRALLOC: LockFlexLayout: baseFormat: 11, yStride: 2448, ySize: 5992704, uOffset: 5992704,  uStride: 2448
2021-09-24 14:05:20.230 20583-20583/? A/DEBUG:     x0  0000000000000000  x1  000000000000501b  x2  0000000000000006  x3  0000000000000008
2021-09-24 14:05:20.230 20583-20583/? A/DEBUG:     x4  fefefefefefefeff  x5  fefefefefefefeff  x6  fefefefefefefeff  x7  7f7f7f7f7f7f7f7f
2021-09-24 14:05:20.230 20583-20583/? A/DEBUG:     x8  0000000000000083  x9  c649dff136bd336f  x10 0000000000000000  x11 fffffffc7ffffbdf
2021-09-24 14:05:20.230 20583-20583/? A/DEBUG:     x12 0000000000000001  x13 00000000614dbf00  x14 000095195c7f9b98  x15 00000f82d8754b2b
2021-09-24 14:05:20.230 20583-20583/? A/DEBUG:     x16 00000079699702c0  x17 00000079698af33c  x18 0000000000000001  x19 0000000000004fd4
2021-09-24 14:05:20.231 20583-20583/? A/DEBUG:     x20 000000000000501b  x21 00000078b0af8928  x22 ffffff80ffffffc8  x23 00000078b0af89e0
2021-09-24 14:05:20.231 20583-20583/? A/DEBUG:     x24 00000078b0af88c0  x25 00000078b0af8900  x26 000000000000000e  x27 0000000000000001
2021-09-24 14:05:20.231 20583-20583/? A/DEBUG:     x28 00000000000000d8  x29 00000078b0af8830
2021-09-24 14:05:20.231 20583-20583/? A/DEBUG:     sp  00000078b0af87f0  lr  00000079698a3e68  pc  00000079698a3e90
2021-09-24 14:05:20.234 20583-20583/? A/DEBUG: backtrace:
2021-09-24 14:05:20.234 20583-20583/? A/DEBUG:     #00 pc 0000000000022e90  /system/lib64/libc.so (abort+116)
2021-09-24 14:05:20.234 20583-20583/? A/DEBUG:     #01 pc 00000000000a01bc  /data/app/com.mrousavy.camera.example-LV6-UFdItuqHIdepcTYoDA==/lib/arm64/libc++_shared.so
2021-09-24 14:05:20.234 20583-20583/? A/DEBUG:     #02 pc 00000000000a03b0  /data/app/com.mrousavy.camera.example-LV6-UFdItuqHIdepcTYoDA==/lib/arm64/libc++_shared.so
2021-09-24 14:05:20.234 20583-20583/? A/DEBUG:     #03 pc 00000000000b4208  /data/app/com.mrousavy.camera.example-LV6-UFdItuqHIdepcTYoDA==/lib/arm64/libc++_shared.so
2021-09-24 14:05:20.234 20583-20583/? A/DEBUG:     #04 pc 00000000000b41a0  /data/app/com.mrousavy.camera.example-LV6-UFdItuqHIdepcTYoDA==/lib/arm64/libc++_shared.so (std::terminate()+36)
2021-09-24 14:05:20.234 20583-20583/? A/DEBUG:     #05 pc 0000000000058ca4  /data/app/com.mrousavy.camera.example-LV6-UFdItuqHIdepcTYoDA==/lib/arm64/libVisionCamera.so
```

### Device

Huawei P10

### VisionCamera Version

master

### Additional information

- [ ] I am using Expo
- [X] I have read the [Troubleshooting Guide](https://mrousavy.github.io/react-native-vision-camera/docs/guides/troubleshooting)
- [X] I agree to follow this project's [Code of Conduct](https://github.com/mrousavy/react-native-vision-camera/blob/main/CODE_OF_CONDUCT.md)
- [X] I searched for [similar issues in this repository](https://github.com/mrousavy/react-native-vision-camera/issues) and found none.

**Comment by mrousavy** [comment]
how tf do I desymbolicate the stacktrace

**Comment by mrousavy** [comment]
Fixed in #455 + [Reanimated v2.3.0-beta.1](https://github.com/software-mansion/react-native-reanimated/releases/tag/2.3.0-beta.1).
